nvtabular/ops/categorify.py

```diff
@@ -829,3 +925,42 @@ def _encode_list_column(original, encoded):
         size=original.size,
         children=(original._column.offsets, encoded),
     )
+
+
+def _hash_bucket(gdf, num_buckets, col):
+    nb = num_buckets[col]
+    if is_list_dtype(gdf[col].dtype):
+        encoded = _encode_list_column(gdf[col], gdf[col].list.leaves.hash_values() % nb)
```

Comment:
I don't think we need the encode_list_column call just yet here -  (and I don't think it will work?). Can you add a unittest w/ list processing to check

Reply:
@benfred list support is not complete yet.  Could you please clarify `I don't think we need the encode_list_column call just yet here` ? where should we define it? in the HashBucket op it was defined as follows, and that's where I got `_encode_list_column` script.

```
for col, nb in num_buckets.items():
            new_col = f"{col}_{self._id}"
            if is_list_dtype(gdf[col].dtype):
                encoded = _encode_list_column(gdf[col], gdf[col].list.leaves.hash_values() % nb)
            else:
                encoded = gdf[col].hash_values() % nb

            new_gdf[new_col] = encoded
```

Doesn't HashBucket work for list columns already?

Reply:
HashBucket works for list columns already - I  meant that the 'encode_list_column' call should be done once, after all the encoding is done (it creates a new list column - which you don't want to operate on here).  

This function should be something like:
```
            if is_list_dtype(gdf[col].dtype):
                encoded =  gdf[col].list.leaves.hash_values() % nb
            else:
                encoded = gdf[col].hash_values() % nb

            new_gdf[new_col] = encoded
```

and the encode_list_column call should be the last thing that happens in the encode function